src/pretix/base/metrics.py

```diff
@@ -0,0 +1,161 @@
+import redis
+
+ns_prefix = "pretix_metrics_"
+
+r = redis.Redis(host=redishost, port=redisport)
```

Comment:
We'll go for ``django_redis.get_redis_connection("redis")`` here. As far as I can see, django-redis does connection pooling, so we don't need it ourselves. Therefore, we can just use it here in global space.